testFile

```diff
@@ -0,0 +1,553 @@
+apicast-operator|threescale-2.8|apicast-operator.v0.2.4|0|0.2.4|||apicast-operator.v0.2.3|apicast-operator.v0.2.4|threescale-2.9
```

Comment:
what are these testFiles? 
If they are useful as a sample could we created and add them in the testdata dir instead?

Reply:
Those were a debugging aid so that sqlite processing was already done. Since we might still do some more debugging, would like to keep them part of the project for now.